spec/src/modules/tracker.js

```diff
@@ -1282,7 +1282,10 @@ describe('ConstructorIO - Tracker', () => {
       ],
       revenue: 123.45,
     };
-    const optionalParameters = { section: 'Products' };
+    const optionalParameters = {
+      order_id: '123938123',
```

Comment:
Should we check for this in the optional params test we have below?

Reply:
> Should we check for this in the optional params test we have below?

Nice catch! Thanks